src/parsers/perf/perfparser.cpp

```diff
@@ -1144,6 +1167,7 @@ class PerfParserPrivate : public QObject
         summaryResult.totalMemoryInKiB = features.totalMem;
 
         eventResult.cpus.resize(features.nrCpusAvailable);
+        disassemblyResult.arch = QString::fromUtf8(features.arch);
```

Comment:
store this in a separate member `m_arch`

Reply:
Do you mean rename the field on m_arch instead arch?

Reply:
no, I mean remove the `disassemblyResult` as a member and instead introduce a separate `m_arch` member, then see below...

Reply:
this is still here, please clean it up

Reply:
It is quite small to be hoisted into separate patch. I just added it to the actualPath patch because the both are related to some extent with cross-platformity.

Reply:
please split it up, commits should be atomic and not do multiple things that are only slightly related